docs/configuration-reference/platforms/aks.md

```diff
@@ -71,9 +71,9 @@ cluster "aks" {
       "key" = "value",
     }
 
-    taints = [
-      "node-role.kubernetes.io/master=NoSchedule",
-    ]
+    taints = {
+      "key1=value1" = "NoSchedule",
```

Comment:
This looks really weird, I think we need to fix this taint story once and for all. Like define an object explicitly saying what is what, instead of map.